Java/examples/src/main/java/com/amazon/randomcutforest/examples/parkservices/RCFCasterExample.java

```diff
@@ -154,7 +154,7 @@ void printResult(BufferedWriter file, ForecastDescriptor result, int current, in
         float[] lowerError = result.getObservedErrorDistribution().lower;
         DiVector rmse = result.getErrorRMSE();
         float[] mean = result.getErrorMean();
-        float[] calibration = result.getCalibration();
+        float[] calibration = result.getIntervalPrecision();
```

Comment:
Should we rename the local variable to intervalPrecision too?

Reply:
yes. Changing.

Reply:
fixed in https://github.com/aws/random-cut-forest-by-aws/pull/393/commits/0ed94fc8c8a6d36839a19c43408b6fb4e2a353fe